<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find primes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="startLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>End Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="threadsLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Threads:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="startSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="endSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="threadsSpinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thread1TextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>150</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="thread1Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread 1 results:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thread2Label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread 2 results:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thread3Label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread 3 results:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="thread4Label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thread 4 results:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thread2TextEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>150</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thread3TextEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>150</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thread4TextEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>150</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
